packages/cel/src/std/logic.test.ts

```diff
@@ -0,0 +1,84 @@
+// Copyright 2024-2025 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { suite, test } from "node:test";
+import * as assert from "node:assert/strict";
+import { performance } from "node:perf_hooks";
+
+import { FuncRegistry } from "../func.js";
+import { addLogic } from "./logic.js";
+
+const registry = new FuncRegistry();
+addLogic(registry);
+
+function durationOf(func: () => any, times: number = 5): number {
```

Comment:
```suggestion
function durationOf(func: () => void, times = 5): number {
```

It's best to avoid any if possible, and it's customary to not annotate types that are trivially inferred.

We don't run the linter on this package yet, but you can run it on select files: `npx biome lint src/std/logic.test.ts` in packages/cel.